src/Features/CSharp/Portable/ChangeSignature/CSharpChangeSignatureService.cs

```diff
@@ -142,6 +147,30 @@ private static int TryGetSelectedIndexFromDeclaration(int position, SyntaxNode m
             return parameters != null ? GetParameterIndex(parameters.Parameters, position) : 0;
         }
 
+        // Find the position to insert the new parameter.
+        // We will insert a new comma and a parameter.
+        protected override int? TryGetInsertPositionFromDeclaration(SyntaxNode matchingNode)
+        {
+            var parameters = matchingNode.ChildNodes().OfType<BaseParameterListSyntax>().SingleOrDefault();
+
+            if (parameters == null)
+            {
+                return null;
```

Comment:
can this happen? if so, when?

Reply:
I don't think so. Removing and will see what happens. :)

Reply:
looks like it's still here.  or did you mean "removed in followup pr"?

Reply:
removed in followup PR.